Mathlib/FieldTheory/Normal.lean

```diff
@@ -39,14 +39,16 @@ class Normal : Prop where
   splits' (x : K) : Splits (algebraMap F K) (minpoly F x)
 #align normal Normal
 
+attribute [instance] Normal.isAlgebraic'
```

Comment:
Why not making it an `instance` from the beginning?

Reply:
I'm not sure this is possible inside a `class` declaration, unless you mean `class Normal extends Algebra.IsAlgebraic`? I suppose I can try that out.

Reply:
Ah sorry, I didn't realize this comes from the definition of `Normal`. No problem then.

Reply:
Actually, I'm already turning it into `extends` and the extra changes are not so bad! So I'll finish that right now.